components/log-viewer-webui/client/vite.config.ts

```diff
@@ -14,6 +14,11 @@ export default defineConfig({
                 target: "http://localhost:3000/",
```

Comment:
Since the comment in L12-13 applies to both routes:
1. Let move the comment to Line 11
2. Let say "target host and port" instead of "target" to improve clarity.

@coderabbitai Let's create an issue to track the effort of serving Fastify routes under "/api". And we should replace this "/query/" proxy entry with "/api/" to ensure the other RESTful APIs are proxied to the backend.